rules_python/whl_test.py

```diff
@@ -60,11 +63,19 @@ def test_mock_whl(self):
     self.assertEqual(wheel.name(), 'mock')
     self.assertEqual(wheel.distribution(), 'mock')
     self.assertEqual(wheel.version(), '2.0.0')
+    if VERSION_TUPLE < (3, 3):
+      expected_deps = ['funcsigs', 'pbr', 'six']
+    else:
+      expected_deps = ['pbr', 'six']
```

Comment:
@duggelz Do you have any tricks to get coverage of both paths?

Reply:
Good point. I think it can be done by mocking `platform.python_version()`. I could add this to this PR or send a separate change after.